common/templates/emails/new_enrollment_email_body.txt

```diff
@@ -0,0 +1,11 @@
+Enrollment received for:
+
+Learner: ${user.username} \
+% if user.get_full_name():
+(${user.get_full_name()})
```

Comment:
I signed up using the client's themed registration form, with matching registration form config, and this offers only the "Full Name" UserProfile field.  So unfortunately, the Django [User.get_full_name()](https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#django.contrib.auth.models.User.get_full_name) method never returns anything.

So I think you need to do [something like this](https://github.com/edx/edx-platform/blob/e8a36957b1f732974260e7b9b42b9c25148b492c/common/djangoapps/xblock_django/user_service.py#L66-L67) instead.